Gemfile.lock

```diff
@@ -11,7 +11,7 @@ GEM
     faraday (2.2.0)
       faraday-net_http (~> 2.0)
       ruby2_keywords (>= 0.0.4)
-    faraday-net_http (2.0.1)
+    faraday-net_http (2.0.2)
```

Comment:
Can we add versions in the gemfile?  gem "faraday", "~> 2.2.0" etc

Reply:
Yes, not sure entirely why but it seems to be best practice. Other projects I'm in do it.